IHP/IDE/SchemaDesigner/Compiler.hs

```diff
@@ -76,12 +76,17 @@ compilePostgresType PTime = "TIME"
 compilePostgresType (PCustomType theType) = theType
 
 compileIdentifier :: _ -> Text
-compileIdentifier identifier = if identifierNeedsQuoting then tshow identifier else identifier
+compileIdentifier identifier = if isHaskellKeyword
```

Comment:
The sql field name should still be the "normal" thing. Only the haskell field should be named differently.

Reply:
Haskell Keywords are fixed in the parser now. Manual SQL changes will now work